lisa/sut_orchestrator/aws/features.py

```diff
@@ -0,0 +1,503 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import base64
+from dataclasses import dataclass
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Type
+
+from assertpy import assert_that
+from dataclasses_json import dataclass_json
+
+from lisa import features, schema, search_space
+from lisa.features.gpu import ComputeSDK
+from lisa.node import Node
+from lisa.operating_system import CentOs, Redhat, Suse, Ubuntu
+from lisa.util import LisaException
+
+if TYPE_CHECKING:
+    from .platform_ import AwsPlatform
+
+from .. import AWS
+from .common import AwsNodeSchema, get_node_context
+
+
+class AwsFeatureMixin:
+    def _initialize_information(self, node: Node) -> None:
+        node_context = get_node_context(node)
+        self._intsance_id = node_context.intsance_id
+
+
+class StartStop(AwsFeatureMixin, features.StartStop):
+    def _stop(self, wait: bool = True) -> None:
+        platform: AwsPlatform = self._platform  # type: ignore
+        ec2_client = platform._ec2_client
+        ec2_client.stop_instances(InstanceIds=[self._intsance_id])
+        ec2_client.get_waiter("instance_stopped").wait()
+
+    def _start(self, wait: bool = True) -> None:
+        platform: AwsPlatform = self._platform  # type: ignore
+        ec2_client = platform._ec2_client
+        ec2_client.start_instances(InstanceIds=[self._intsance_id])
+        ec2_client.get_waiter("instance_running").wait()
+
+    def _restart(self, wait: bool = True) -> None:
+        platform: AwsPlatform = self._platform  # type: ignore
+        ec2_client = platform._ec2_client
+        ec2_client.reboot_instances(InstanceIds=[self._intsance_id])
+        ec2_client.get_waiter("instance_running").wait()
```

Comment:
Use the `wait` argument to decide whether waiting in this method.

Reply:
Updated, thanks

Reply:
The _start/_stop should use the `wait` argument as well.

Reply:
Fixed.